components/text-utilities/text-utilities.css

```diff
@@ -0,0 +1,19 @@
+@import '../../homepage/homepage.css';
+@import '../document.css';
+
+.column-flex{
+    flex-direction: column;
+}
+
+.biggest-heading{
+    font-size: 2rem;
+}
+
+.gray-text{
+    color: var(--gray);
+    font-size: larger;
```

Comment:
I think Kanishk had mentioned not to use larger, instead you could use variables or even utility classes for text sizes